### PR TITLE
GYSR token update

### PR DESCRIPTION
**Pull request by devinaconley**
This PR includes an update to the GYSR token logo and info. It also adds the Polygon (PoS) mapping of the same token

### CODE REVIEW

**Comment by devinaconley** [comment]
Original PR here with discussion/explanation on holder count
https://github.com/trustwallet/assets/pull/5081

File: blockchains/polygon/assets/0xc48F61a288A08F1B80c2edd74652e1276B6A168c/info.json [hunk 1]
```diff
@@ -0,0 +1,52 @@
+{
+    "name": "GYSR (PoS)",
```
Comment (Cryptocool1) [inline blockchains/polygon/assets/0xc48F61a288A08F1B80c2edd74652e1276B6A168c/info.json:2]: please change name to  "GYSR"

Comment (devinaconley) [reply blockchains/polygon/assets/0xc48F61a288A08F1B80c2edd74652e1276B6A168c/info.json]: fixed :+1:

**Comment by devinaconley** [comment]
Thanks @Cryptocool1 - just fixed the bridged token name

**Pull request merged**